tenant-isolation-config-service-api/src/main/proto/org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_config_service.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantisolation.config.service.v1;
+
+option java_multiple_files = true;
+
+import "org/hypertrace/tenantisolation/config/service/v1/tenant_isolation_group_config.proto";
+
+service TenantIsolationConfigService{
```

Comment:
The naming here feels a bit off (applies to whole PR). It's not really _isolation_ so much as defining how to partition tenants which may or may not be isolated, right?

Reply:
yea not very happy with the naming here - how does TenantPartitionGroupConfigService sound ?

Reply:
I'd probably go even simpler to just `TenantPartitioningConfigService`